### PR TITLE
[APIView] Change display of multiple guidelines

### PR DESCRIPTION
**Pull request by AlitzelMendez**
Issue: https://github.com/Azure/azure-sdk-tools/issues/11098

## Multiple guidelines
image

## TODO
- [ ] Add couple of tests for scenarios in where there is none, one and multiples guidelines ids

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs [hunk 1]
```diff
@@ -100,10 +100,17 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
...
-                                foreach (var id in comment.RuleIds)
+
+                                if (comment.RuleIds.Count > 0)
                                 {
-                                    commentText.AppendLine($"See: https://azure.github.io/azure-sdk/{id}");
+                                    commentText.AppendLine("**Guidelines**");
+                                    foreach (var ruleId in comment.RuleIds)
+                                    {
+                                        commentText.AppendLine();
+                                        commentText.AppendLine($"https://azure.github.io/azure-sdk/{ruleId}");
```
[the remark this answers is not in the document]
Comment (AlitzelMendez) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:110]: thanks, this was completely intentional, we don't want the extra space provided by a list with `-`

File: src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs [hunk 2]
```diff
@@ -100,10 +100,17 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                                     commentText.AppendLine();
                                     commentText.AppendLine();
                                 }
-                                foreach (var id in comment.RuleIds)
+
+                                if (comment.RuleIds.Count > 0)
                                 {
-                                    commentText.AppendLine($"See: https://azure.github.io/azure-sdk/{id}");
+                                    commentText.AppendLine("**Guidelines**");
```
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:106]: will they always be guidelines?

Comment (AlitzelMendez) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:106]: no, not always, we are just adding this when there are `RuleIds.Count > 0` which are the ruleIds of the guidelines 

or do you mean in plural, like having `Guideline` vs `Guidelines`?

Comment (tjprescott) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:106]: Links will always be guidelines. Examples and memories don't make sense to have links since they aren't published anywhere.

Comment (tjprescott) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:106]: Even with a single link, I think "Guidelines" makes sense...

Comment (AlitzelMendez) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:106]: oh! sorry Mariana I didn't understand your question! thanks Travis for jumping in!!!
